<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Syrgas-MBP.lan" failures="1" tests="3" name="TestNGExample" time="0.051" errors="0" timestamp="2023-01-21T02:32:05 CST">
  <testcase classname="com.automation.test.CartPageTest" name="verifyUserCanRemoveItemFromCart" time="0.001">
    <failure type="java.lang.RuntimeException" message="Test Failed!">
      <![CDATA[java.lang.RuntimeException: Test Failed!
at com.automation.test.CartPageTest.verifyUserCanRemoveItemFromCart(CartPageTest.java:9)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- verifyUserCanRemoveItemFromCart -->
  <testcase classname="com.automation.test.CartPageTest" name="verifyUserCanAddItemIntoCart" time="0.0">
    <skipped/>
  </testcase> <!-- verifyUserCanAddItemIntoCart -->
  <testcase classname="com.automation.test.CartPageTest" name="verifyUserCanChangeQuantityOfItemOnCart" time="0.0">
    <skipped/>
  </testcase> <!-- verifyUserCanChangeQuantityOfItemOnCart -->
</testsuite> <!-- TestNGExample -->
